Update README with image details (#7)
containers/image_build: README.md

# Image Build

Monorepo menagerie of container images and associated build automation

## Podman / Buildah / Skopeo

## Overview

> [comment]: <> (***ATTENTION*** ***WARNING*** ***ALERT*** ***CAUTION*** ***DANGER***)

Reviewer: Where's Will Robinson when you need him?

[comment]: <> ()
[comment]: <> (ANY changes made below, once committed/merged must)
[comment]: <> (be manually copy/pasted -in markdown- into the description)
[comment]: <> (field on Quay at the following locations, where * represents podman|buildah|skopeo:)
[comment]: <> ()
[comment]: <> (https://quay.io/repository/containers/*)
[comment]: <> (https://quay.io/repository/*/stable)
[comment]: <> (https://quay.io/repository/*/testing)
[comment]: <> (https://quay.io/repository/*/upstream)
[comment]: <> ()
[comment]: <> (***ATTENTION*** ***WARNING*** ***ALERT*** ***CAUTION*** ***DANGER***)

These directories contain the Containerfiles necessary to create the images housed on

> quay.io under the their namespace in addition to the 'containers' namespace.  These

Reviewer: ```suggestion
quay.io under their namespace in addition to the 'containers' namespace.  These
```

> images public and can be pulled without credentials.  These container images are secured and the

Reviewer: ```suggestion
images are public and can be pulled without credentials.  These container images are secured and the
```

> resulting containers can run safely with or without privleges.

Reviewer: ```suggestion
resulting containers can run safely with or without privileges.
```

The container images are built using the latest Fedora and then the respective tools are installed.
The `$PATH` in the container images is set to the default provided by Fedora.  Neither the

> `$ENTRYPOINT` and the `$WORKDIR` variables are set within these container images, and as

Reviewer: ```suggestion
`$ENTRYPOINT` nor the `$WORKDIR` variables are set within these container images, and as
```

such they default to `/`.

The container images are tagged as follows, where `*` represents either `podman`, `buildah`

> or `skopeo`:

Reviewer: This is an IDK question.  Is Brent cooking up a new machine image and should, or maybe will be, included here too?

Author: That's my hope for the future.  It makes a lot of sense for several reasons to put both the image content and build automation close.

  * `quay.io/containers/*:<version>` and `quay.io/*/stable:<version>` -
    These images are built daily.  They are intended to contain an unchanging

>     and stable version of podman. For the most recent `<version>` tags (`vX`,

Reviewer: ```suggestion
    and stable version of their container image. For the most recent `<version>` tags (`vX`,
```

    `vX.Y`, and `vX.Y.Z`) the image contents will be updated daily to incorporate

>     (especially) security upgrades.

Reviewer: on the fence, upgrades or updates?  I think updates, but I'll let you make the call.

Author: Yeah updates is probably better.  Thanks.

resulting containers can run safely with or without privileges.

The container images are built using the latest Fedora and then the respective tools are installed.
The `$PATH` in the container images is set to the default provided by Fedora.  Neither the
`$ENTRYPOINT` nor the `$WORKDIR` variables are set within these container images, and as
such they default to `/`.

The container images are tagged as follows, where `*` represents either `podman`, `buildah`
or `skopeo`:

  * `quay.io/containers/*:<version>` and `quay.io/*/stable:<version>` -
    These images are built daily.  They are intended to contain an unchanging
    and stable version of their container image. For the most recent `<version>` tags (`vX`,
    `vX.Y`, and `vX.Y.Z`) the image contents will be updated daily to incorporate
    (especially) security updates.
  * `quay.io/containers/*:latest` and `quay.io/*/stable:latest` -
    Built daily using the same `Containerfile` as above.  The tool versions
    will remain the "latest" available in Fedora.
  * `quay.io/*/testing:latest` - This image is built daily, using the
    latest tooling version available in the Fedora `updates-testing` repository.

>   * `quay.io/*/upstream:latest` - This image is built daily using the latest

Reviewer: IDK, but I think it might be good to bold the backticked image names to make them stand out a bit more.  Perhaps?

```suggestion
  * **`quay.io/*/upstream:latest`** - This image is built daily using the latest
```
If you do it here, do it for the others above.

Author: Github already renders them in `a stand-out way`, I'm not sure the **`additional bolding`** would add very much.  I won't be able to make that change here, but if you still feel it's needed in the future, please feel free to open a PR.  I've rigged up CI to automatically avoid testing anything for README updates :grinning:
